data/json/recipes/recipe_deconstruction.json

```diff
@@ -5861,6 +5861,14 @@
     "activity_level": "MODERATE_EXERCISE",
     "time": "12 s",
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ [ "scrap", 2 ] ] ]
+  },
+  {
+    "result": "can_food",
+    "type": "uncraft",
+    "activity_level": "MODERATE_EXERCISE",
+    "time": "12 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
```

Comment:
Please delete this. This generates mass. ``scrap`` is 50g, medium can is 70g, small can is 40g...

Reply:
Resolved. Will audit cans at some point to give them correct mass, right after I make chili.